### PR TITLE
remove roms, add import script

### PR DESCRIPTION
**Pull request by christopherhesse**

### CODE REVIEW

**Comment by pzhokhov** [comment]
two comments: 
1) why do we include libale.so into the git repo? 
2) the md5 checksum check - is that to make sure roms are not accidentally broken by a bad download, or to make sure noone tampered with them? If the latter (which I think is a good idea, given that now those files are coming from external source), then I think we should rather use sha256 checksums. 
Otherwise - LGTM

**Comment by christopherhesse** [comment]
1) looks like so files were not gitignored
2) They could be converted to sha checksums by downloading files that match the existing md5s, then hashing them and replacing the hashes, but this seems odd.  The reason they are md5s is that this file is copied from ALE: https://github.com/mgbellemare/Arcade-Learning-Environment/blob/master/md5.txt

File: atari_py/package_data.txt [hunk 1]
```diff
@@ -367,69 +368,7 @@ ale_interface/src/os_dependent/SettingsUNIX.hxx
 ale_interface/src/os_dependent/SettingsWin32.cxx
 ale_interface/src/os_dependent/SettingsWin32.hxx
 ale_interface/src/os_dependent/module.mk
+atari_roms/tetris.bin
```
Comment (pzhokhov) [inline atari_py/package_data.txt]: is that ok if we keep tetris? I understand we want it for tests, but would not get us in just as much of a legal pickle as the rest of the roms?

**Comment by pzhokhov** [comment]
LGTM! Yeah, if the md5 files are not generated by us, let's not worry about recomputing the checksums.

File: atari_py/package_data.txt (hunk 1 above)
Comment (christopherhesse) [reply atari_py/package_data.txt]: This is a non-commercial rom as mentioned here: https://github.com/openai/atari-py/pull/62/files#diff-04c6e90faac2675aa89e2176d2eec7d8R37

**Comment by christopherhesse** [comment]
Thanks!

**Pull request merged**